lib/package-manager.coffee

```diff
@@ -4,6 +4,8 @@ _ = require 'underscore-plus'
 Q = require 'q'
 semver = require 'semver'
 url = require 'url'
+apm = require 'apm'
```

Comment:
Sounds good to me. 
I'll open an issue on apm to support the `--compatible` option in the `view` command.